### PR TITLE
GUVNOR-2923: [Guided Decision Table] Audit log is not paginated

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/GUVNOR-2923

### CODE REVIEW

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/auditlog/AuditLogViewImpl.java [hunk 1]
```diff
@@ -145,122 +133,111 @@ public void show() {
...
+        events.addStyleName(Styles.TABLE);
+
+        dlp = new ListDataProvider<AuditLogEntry>() {
 
-        dlp = new ListDataProvider<AuditLogEntry>();
-        dlp.addDataDisplay( events );
+            @Override
+            public void setList(final List<AuditLogEntry> listToWrap) {
+                super.setList(listToWrap);
+                cellTablePagination.rebuild(pager);
```
Comment (manstis) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/auditlog/AuditLogViewImpl.java]: Rebuild BS3's "pager" when the ```DataListProvider``` content is set...

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/auditlog/AuditLogViewImplTest.java [hunk 2]
```diff
@@ -43,43 +46,53 @@
...
         view.show();
 
-        verify( view,
-                times( 1 ) ).refreshDataProvider();
+        verify(view,
+               times(1)).refreshDataProvider();
+    }
+
+    @Test
+    public void checkPaginationIsSynchronizedWithDataListProvider() {
```
Comment (manstis) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/auditlog/AuditLogViewImplTest.java]: And... a test.

**Comment by manstis** [comment]
@jomarko Care to review? I refactored a little; but nothing too serious.

**Comment by jomarko** [comment]
@manstis sure. I would like to have a look. I will put comment here when I am done.

**Comment by jomarko** [comment]
Tried locally and pagination works now. I have also nothing against the code changes. Thanks @manstis for solving this issue in such short time.

**Pull request merged**